docs/conf.py

```diff
@@ -125,6 +130,25 @@ def noop_decorator(*args, **kwargs):
     "sidebarwidth": 270,
     "collapse_navigation": False,
     "navigation_depth": 4,
+    "show_toc_level": 2,
+    "github_url": "https://github.com/modin-project/modin",
+    "icon_links": [
+        {
+            "name": "PyPI",
+            "url": "https://pypi.org/project/modin",
+            "icon": "fab fa-python",
+        },
+        {
+            "name": "Join the Slack",
+            "url": "https://modin.org/slack.html",
+            "icon": "fab fa-slack",
+        },
+        {
+            "name": "Mailing List",
+            "url": "https://groups.google.com/forum/#!forum/modin-dev",
+            "icon": "fas fa-envelope-square",
+        },
+    ],
```

Comment:
shouldn't discourse and conda-forge also be here?

Reply:
Added discource and conda-forge to the icon-links section. Unfortunately icon-links only support [Font Awesome](https://fontawesome.com/) icons. I couldn't find an exact match for anaconda, so it's currently just a circle with a break. There's not a good workaround for this.